docs/wip/exchange-onboarding.md

```diff
@@ -0,0 +1,352 @@
+      #walletListen = "127.0.0.1:8090";
+      #ekgListen = "127.0.0.1:8000";
+      stateDir = "./state-wallet-mainnet";
+      topologyFile = ./exchange-topology.yaml;
+    
+      ## See https://downloads.haskell.org/~ghc/8.0.2/docs/html/users_guide/runtime_control.html#running-a-compiled-program
+      #ghcRuntimeArgs = "-N2 -qg -A1m -I0 -T";
+    
+      ## Primarily used for troubleshooting.
+      #additionalNodeArgs = "";
+    }
+
+The rest of this document will assume the above configuration file. Please
+alter any commands for example related to `stateDir` path to reflect your
+setup.
+
+You will also need to add the following `exchange-topology.yaml` file to use the
+private relays:
+
+    TODO: private relays should be inserted here in document given to exchanges
+
+
+## Build and run in the nix store
+
+By default the wallet's local state goes in
+`./state-wallet-mainnet`. If you prefer to have this sensitive data
+outside the git repository, see the FAQ about supported custom
+configuration settings (See section 2.4.1).
+
+Build the wallet and generate the shell script to connect to
+mainnet (use `connectScripts.stagingWallet` for testnet)
+
+    nix-build -A connectScripts.mainnetWallet -o "./launch_$(date -I)_$(git rev-parse --short HEAD)"
+
+After the build finishes the generated connection script is
+available as a symlink called `./launch_2018-01-30_0d4f79eea`, or
+similar. Run that symlink as a script to start the wallet.
+
+## Build and run a docker image
+
+Follow the above instructions for customization and dependencies. To build a docker
+container and import the image run
+(use `connectScripts.stagingWallet` for testnet):
+
+    docker load < $(nix-build -A dockerImages.mainnetWallet))
+
+This will create an image `cardano-container-mainnet:latest`
+(or `cardano-container-staging:latest` for testnet)
+
+After this image is built, it can be used like any other docker image being pushed
+into a registry and pulled down using your preferred docker orchestration tool.
+
+The image can be ran using the following:
+
+    docker run --name cardano-mainnet-wallet --rm -it -p 127.0.0.1:8090:8090 -p 127.0.0.1:8000:8000 -v state-wallet-mainnet:/wallet cardano-container-mainnet:latest
+
+The above command will create a docker volume named `state-wallet-mainnet` and will mount
+that to /wallet. Note: if no volume is mounted to `/wallet` the container startup
+script will refused to execute `cardano-node` and the container will exit.
+
+The location of `/wallet` cannot be changed, but you can mount any kind of volume
+you want in that directory that docker supports.
+
+Note that if you give this a different name than is specified above, use the
+name you used for any future docker commands in examples in the document.
+
+# Migrating from V0 to V1 API
+
+# Usage FAQs
+
+## What are recommended hardware/software requirements for exchange wallets?
+
+RAM: 8 GB for building, 4 GB for running
+CPU: ???
+Disk: SSD recommended of ??? size
```

Comment:
100 GB, with the option to allocate more space in coming years. Sound good, @cleverca22?

@disassembler, please add a FAQ about the expected log rotation/expunging behavior with the documented configuration.